src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.cs

```diff
@@ -110,8 +110,8 @@ private void UpdateColorScheme(bool themeChanged = false)
         {
             AssertIsForeground();
 
-            // Simply return if we were queued to run during shutdown or the user is in High Contrast mode.
-            if (_isDisposed || SystemParameters.HighContrast)
+            // Simply return if we were queued to run during shutdown.
+            if (_isDisposed)
```

Comment:
Should we explicitly special case `if (SystemParameters.HighContrast)` and do precisely what's needed in that case and then return, as opposed to letting the correct behavior fall out implicitly below?

Reply:
Everything else is a safe operation and things that would need to be done eventually. Migrate from useEnhanced to ColorScheme option